applications/StructuralMechanicsApplication/custom_elements/solid_elements/small_displacement_mixed_strain_displacement_element.h

```diff
@@ -0,0 +1,766 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:         BSD License
+//                   license: StructuralMechanicsApplication/license.txt
+//
+//  Main authors:    Alejandro Cornejo
+//
+//
+
+#pragma once
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "includes/element.h"
+#include "utilities/integration_utilities.h"
+
+// Application includes
+#include "structural_mechanics_application_variables.h"
+#include "utilities/geometry_utilities.h"
+
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+    /// The definition of the index type
+    using IndexType = std::size_t;
+
+    /// The definition of the sizetype
+    using SizeType = std::size_t;
```

Comment:
Note that these would apply to the entire Kratos namespace. Put them into the class type definitions.